inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -439,7 +439,15 @@ private function is_mobile(): bool {
 	 * @return bool
 	 */
 	private function is_home( string $url ): bool {
-		return untrailingslashit( $url ) === untrailingslashit( home_url() );
+		/**
+		 * Filters the home url.
+		 *
+		 * @since 3.11.4
+		 *
+		 * @param bool  $is_home boolean return if current url is_home.
+		 * @param string  $url url of current page.
+		 */
+		return apply_filters( 'rocket_rucss_is_home_url', ( untrailingslashit( $url ) === untrailingslashit( home_url() ) ), $url );
```

Comment:
We should just filter the `home_url()` value here, and then apply the rest of the logic. Something like:
```
$home_url = apply_filters( 'rocket_rucss_home_url', home_url() );

return untrailingslashit( $url ) === untrailingslashit( $home_url );
```